tests/data/parser.json

```diff
@@ -1,1963 +1,2424 @@
 [
   {
-    "test": "0001",
+    "test": "0001", 
```

Comment:
Would it be possible to avoid adding this trailing whitespace?  It's adding a lot of noise to the diff.